includes/wikia/WikiaUpdater.php

```diff
@@ -52,6 +52,8 @@ public static function update( DatabaseUpdater $updater ) {
 			array( 'dropIndex', 'wall_related_pages', 'comment_id_idx',  $dir . 'patch-wall_related_pages-drop-comment_id_idx.sql', true ), // SUS-3096
 			array( 'dropIndex', 'wall_related_pages', 'page_id_idx_2',  $dir . 'patch-wall_related_pages-drop-page_id_idx_2.sql', true ), // SUS-3096
 			array( 'dropIndex', 'video_info', 'added_at',  $dir . 'patch-video_info-drop-added_at_idx.sql', true ), // SUS-4297
+			array( 'dropIndex', 'image', 'img_usertext_timestamp', $dir . 'patch-image-drop-usertext-index.sql', true ),
```

Comment:
@mszabo-wikia why are we changing these indexes?

Reply:
they are not used according to query-digest, so we should replace them with one that can be used for the new queries (to delete entries by user ID)